### ISSUE TITLE
Currently Pax Runner --snapshot does not download the correct snapshot versions (equinox+felix) [PAXRUNNER-266]

### ISSUE THREAD

**Issue by ops4j-issues** [ops4j/org.ops4j.pax.runner#291]
**[Toni Menzel](https://ops4j1.jira.com/secure/ViewProfile.jspa?accountId=557058:e7e50a9b-1dd6-45df-b446-7b0e65306e2b)** created **[PAXRUNNER-266](https://ops4j1.jira.com/browse/PAXRUNNER-266)** 

In Felix case:<br /> doing pax-run.sh --definitionURL=[http://scm.ops4j.org/repos/ops4j/projects/pax/runner-repository/org/ops4j/pax/runner/platforms/felix/1.9.0-SNAPSHOT/felix-1.9.0-SNAPSHOT.xml](http://scm.ops4j.org/repos/ops4j/projects/pax/runner-repository/org/ops4j/pax/runner/platforms/felix/1.9.0-SNAPSHOT/felix-1.9.0-SNAPSHOT.xml)<br /> --version=SNAPSHOT<br /> works

but <br /> pax-run.sh --definitionURL=mvn:[http://scm.ops4j.org/repos/ops4j/projects/pax/runner-repository!org.ops4j.pax.runner.platforms/felix//xml](http://scm.ops4j.org/repos/ops4j/projects/pax/runner-repository!org.ops4j.pax.runner.platforms/felix//xml)<br /> --version=SNAPSHOT

does not.

URL "mvn:[http://scm.ops4j.org/repos/ops4j/projects/pax/runner-repository!org.ops4j.pax.runner.platforms/felix//xml](http://scm.ops4j.org/repos/ops4j/projects/pax/runner-repository!org.ops4j.pax.runner.platforms/felix//xml)" resolves to:

```java
<platform>

  <name>Felix 1.8.0</name>
  <system>mvn:org.apache.felix/org.apache.felix.main/1.8.0</system>

  <packages>org.osgi.framework; version=1.4.0, org.osgi.framework.hooks.service; version=1.4.0, org.osgi.service.packageadmin; version=1.2.0, org.osgi.service.startlevel; version=1.1.0, org.osgi.service.url; version=1.0.0, org.osgi.util.tracker; version=1.3.3</packages>

  <profile name="minimal" default="true">
    <bundle>
      <name>org.osgi.compendium (4.1.0)</name>
      <url>mvn:org.osgi/org.osgi.compendium/4.1.0</url>
    </bundle>
  </profile>

  <profile name="tui" extends="minimal">
    <bundle>
      <name>org.apache.felix.shell (1.2.0)</name>
      <url>mvn:org.apache.felix/org.apache.felix.shell/1.2.0</url>
    </bundle>
    <bundle>
      <name>org.apache.felix.shell.tui.plugin (1.2.0)</name>
      <url>mvn:org.apache.felix/org.apache.felix.shell.tui/1.2.0</url>
    </bundle>
  </profile>

</platform>
```

Which is not correct.

---

**Fixed in:** (none)
Votes: 0, Watches: 0

**Comment by ops4j-issues** [comment]
**[Toni Menzel](https://ops4j1.jira.com/secure/ViewProfile.jspa?accountId=557058:e7e50a9b-1dd6-45df-b446-7b0e65306e2b)** commented

snapshots in repo must be versioned as "LATEST" (folder + xml file).<br /> This is what Pax URL mvn handler is looking for after successfully selecting a (concrete) snapshot version from maven-metadata.xml

**Comment by ops4j-issues** [comment]
**[Toni Menzel](https://ops4j1.jira.com/secure/ViewProfile.jspa?accountId=557058:e7e50a9b-1dd6-45df-b446-7b0e65306e2b)** commented

even though, i would like to change this to Guillaume's instinctive syntax.

**Comment by ops4j-issues** [comment]
**[Alin Dreghiciu](https://ops4j1.jira.com/secure/ViewProfile.jspa?accountId=557058:a786db11-2013-41fb-af1d-ca5eb8f74aa0)** commented

Released as part of 1.0.0
